assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/exporter/SW360ExporterTest.java

```diff
@@ -105,5 +108,6 @@ public void testExporter() throws IOException {
         assertThat(csvFile).exists();
         CSVParser csvParser = SW360TestUtils.getCsvParser(csvFile);
         assertThat(csvParser.getRecords().size()).isEqualTo(1);
+        verify(releaseClientAdapterMock, atLeast(1)).downloadAttachment(any(), any(),any() ,any());
```

Comment:
Maybe some concrete expected values can be used instead of the any()s to make the test more meaningful.